app/scripts/directives/dropdownItem.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+angular.module('openshiftConsole').directive('dropdownItem', ['$compile', function ($compile) {
+  return {
+    restrict: 'E',
+    transclude: true,
+    scope: {
+      action: '&',
+      enabled: '@'
+    },
+    link: function ($scope, $element, $attrs, ctrl, transcludeFn) {
+      var currentElement = $element;
+
+      function onAttrChange() {
+        var template = $scope.enabled === 'true' ?
```

Comment:
I think it's expecting the `?` on the same line to avoid potential problems with automatic semicolon insertion in JavaScript. (Not a problem here, but could be for `return` statements and a few other places.) I usually do something like

```js
condition ? thenExpr
          : elseExpr;
```

which I think the linter allows